rpc/rpcserver/server.go

```diff
@@ -1657,10 +1657,16 @@ func (t *ticketbuyerServer) SetMaxPriceRelative(ctx context.Context, req *pb.Set
 	if err != nil {
 		return nil, err
 	}
-	if req.MaxPriceRelative < 0 {
+	maxPriceRelative, err := dcrutil.NewAmount(req.MaxPriceRelative)
```

Comment:
maxPriceRelative is NOT a dcrutil.Amount, it's a perecentage so it must be left as a float64